src/file-uploader/__tests__/get-accepted-files.test.js

```diff
@@ -0,0 +1,41 @@
+import { buildFiles, buildFile } from '../../test/utils'
+import getAcceptedFiles from '../src/utils/get-accepted-files'
+
+describe('getAcceptedFiles', () => {
```

Comment:
Note: These tests are not complete (I started here and quickly realized it will build up to be a huge changeset, so the rest are on another branch)